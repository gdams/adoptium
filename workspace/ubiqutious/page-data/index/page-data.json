{"componentChunkName":"component---src-pages-index-tsx","path":"/","result":{"data":{"locales":{"edges":[]},"mostRecentLts":{"version":21}},"pageContext":{"language":"en","i18n":{"language":"en","languages":["en","en-GB","de","es","fr","zh-CN"],"defaultLanguage":"en","generateDefaultLanguagePage":false,"routed":false,"originalPath":"/","path":"/"}}},"staticQueryHashes":["764694655"],"slicesMap":{"adoptiumNews":"adoptiumNews","banner":"banner","footer":"footer","navbar":"navbar"}}