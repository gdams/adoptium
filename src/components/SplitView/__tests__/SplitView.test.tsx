import React from 'react';
import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest'
import { SplitView, SplitViewLeft, SplitViewRight } from '../index';

describe('SplitView component', () => {
    it('should render correctly', () => {
        const { container } = render(
            <SplitView>
                <SplitViewLeft>
                    <div>Left</div>
                </SplitViewLeft>
                <SplitViewRight>
                    <div>Right</div>
                </SplitViewRight>
            </SplitView>
        );

        expect(container).toMatchSnapshot();
    });
});
