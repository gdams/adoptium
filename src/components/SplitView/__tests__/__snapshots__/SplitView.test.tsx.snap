// Vitest Snapshot v1

exports[`SplitView component > should render correctly 1`] = `
<div>
  <table>
    <tbody>
      <tr>
        <td
          class="split-view-left p-2"
        >
          <div>
            Left
          </div>
        </td>
        <td
          class="split-view-right p-2"
        >
          <div>
            Right
          </div>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
